### ISSUE TITLE
feature request: support for "catch all" error in val-default-errors

### ISSUE THREAD

**Issue by david-bulte** [Ninja-Squad/ngx-valdemort#264]
Hi, I was wondering what you think of some sort of "catch all" error. Then you could have something like so in your default errors:

```
   <ng-template valError="*" let-label let-error="error">{{ errors | translate }}</ng-template>
```

That way you could don't have to list all possible errors in the default errors component.

**Comment by jnizet** [comment]
Hi @david-bulte 

Thanks for your issue. Even though we think, that, to have a better UX, specific error messages are better than a generic message, we also agree that this use-case can be useful, at least to display a generic error rather than nothing at all if an unexpected or forgotten validation error happens on a control.

We think however that it would probably be clearer and simpler to introduce an additional directive. The code would look like this, inside `<val-errors>` and inside `<val-default-errors>`:

```html
<ng-template valDefaultError let-label><span>{{ label || 'this field' }} is invalid</span></ng-template>
```

If the `valDefaultError` is specified inside a `val-errors`, it would overwrite the default `valDefaultError` specified inside `val-default-errors`.

Does that suit your expectations?

I think this should also address your request in #265 (see discussion in this issue for details)

**Comment by jnizet** [comment]
@david-bulte I've published version 5.1.0-beta.1. It would be nice if you could try it and tell us if it works fine for you.

**Comment by david-bulte** [comment]
Hi @jnizet It works great! Thanks!
